collector/mongodb_collector.go

```diff
@@ -176,6 +182,20 @@ func (exporter *MongodbCollector) collectCollectionStatus(session *mgo.Session,
 	}
 }
 
+func (exporter *MongodbCollector) collectProfileStatus(session *mgo.Session, ch chan<- prometheus.Metric) {
+	database_names, err := session.DatabaseNames()
```

Comment:
use camelCase syntax please :)

Reply:
agreed, but there already code like this, I'm correcting also others occurences.